e2e-tests/portal/cypress/integration/journeys/admin/create-user/update-a-user.spec.js

```diff
@@ -19,6 +20,19 @@ context('Admin user updates an existing user', () => {
     cy.removeUserIfPresent(userToUpdate, ADMIN);
   });
 
+  it('Go to users and back', () => {
+    // Login and go to the dashboard
+    cy.login(ADMIN);
+    cy.url().should('include', '/dashboard/deals');
+    header.users().click();
+
+    // Cancel
+    users.cancel().click();
+
+    // Back to the dashboard
+    cy.url().should('include', '/dashboard/deals');
+  });
+
```

Comment:
Can we add a test that edits the user's password (on their behalf) and then we log in using the new password?

Reply:
Added

Reply:
I can't see it in this file? Unless it's somewhere else?